tasks/install.yaml

```diff
@@ -0,0 +1,30 @@
+- block:
+
+  - name: Add MariaDB repo key
+    apt_key:
+      url: https://mariadb.org/mariadb_release_signing_key.asc
```

Comment:
If `mariadb.mirror` is a config option, this should also be a config option. We should also make sure we can add support for other distros later.

```yaml
mariadb:
  repository:
    apt:
      key_url: https://...
      repository: deb http://...
```